static/js/wicd.js

```diff
@@ -106,14 +106,37 @@ function scan_networks() {
   $.getJSON( "scan", refresh_networks );
 }
 
-function connect_to_network() {
-  var network_id = $(this).data('id');
-  showAlert('Connecting...', 'info');
-  $.getJSON( "connect/" + network_id, function(data) {
-    showAlert('Please wait (' + data.data + ')', 'info');
-    showAlert('Connecting to ' + data.data, 'info');
-    watch_connection_status();
-  } );
+function network_status_precheck(){
```

Comment:
*quadrupled, might I suggest a [linter for your prose](https://atom.io/packages/linter-write-good)? :D